programs/uxd/src/utils/calculate_credix_lp_depository_target_amount.rs

```diff
@@ -1,21 +1,39 @@
+use super::calculate_depositories_sum_value;
+use super::calculate_depositories_target_redeemable_amount;
 use crate::state::controller::Controller;
 use crate::state::credix_lp_depository::CredixLpDepository;
 use crate::state::identity_depository::IdentityDepository;
 use crate::state::mercurial_vault_depository::MercurialVaultDepository;
+use crate::utils::checked_convert_u128_to_u64;
 use crate::utils::DepositoryInfoForTargetRedeemableAmount;
 use anchor_lang::prelude::*;
 
-use super::calculate_depositories_target_redeemable_amount;
-
 pub fn calculate_credix_lp_depository_target_amount(
```

Comment:
this function needs a cleanup, but this could be done at a different PR later if we want